### PR TITLE
Add the Core / Extension attribute

### PR DESCRIPTION
**Pull request by balhar-jakub**
Signed-off-by: GitHub <noreply@github.com>

### CODE REVIEW

**Review by janan07 — APPROVED** [review]
Reviewed and refactored for langauge clarity.

File: PROJECTS.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- API Mediation Layer
+  - Gateway Service
+  - Discovery Service
+  - API Catalog Service
+  - Caching Service
+  - Files
+  - Datasets
+- CLI
+- Explorer
+- ZSS
```
Comment (balhar-jakub) [inline PROJECTS.md:64]: Add ZIS

File: PROJECTS.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  - JES Explorer
+  - MVS Explorer
+  - SSH App
+  - TN3270 App
+- API Mediation Layer
+  - Gateway Service
+  - Discovery Service
+  - API Catalog Service
+  - Caching Service
+  - Files
```
Comment (balhar-jakub) [inline PROJECTS.md:20]: Claim visibly deprecation

File: PROJECTS.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  - Datasets
+- CLI
+- Explorer
```
Comment (balhar-jakub) [inline PROJECTS.md:63]: Subbullets
 Zowe Explorer VS Code Extensions 
 Extender APIs (NPM package)

File: PROJECTS.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- ZSS
+
```
Comment (balhar-jakub) [inline PROJECTS.md:65]: Add Zowe SDK at least somewhere

**Pull request merged**